Formula/podman.rb

```diff
@@ -17,23 +17,43 @@ class Podman < Formula
   depends_on "go-md2man" => :build
   depends_on "qemu" if Hardware::CPU.intel?
 
+  resource "gvproxy" do
+    url "https://github.com/containers/gvisor-tap-vsock/archive/v0.1.0.tar.gz"
+    sha256 "e1e1bec2fc42039da1ae68d382d4560a27c04bbe2aae535837294dd6773e88e0"
+  end
+
   def install
-    system "make", "podman-remote-darwin"
-    bin.install "bin/darwin/podman"
+    os = "darwin"
+    on_linux do
+      os = "linux"
+    end
+
+    system "make", "podman-remote-#{os}"
+    on_macos do
+      bin.install "bin/#{os}/podman" => "podman-remote"
+    end
+    on_linux do
+      bin.install "bin/podman-remote"
+    end
+
+    resource("gvproxy").stage do
+      system "make"
+      bin.install "bin/gvproxy"
+    end
 
-    system "make", "install-podman-remote-darwin-docs"
-    man1.install Dir["docs/build/remote/darwin/*.1"]
+    system "make", "install-podman-remote-#{os}-docs"
+    man1.install Dir["docs/build/remote/#{os}/*.1"]
 
     bash_completion.install "completions/bash/podman"
     zsh_completion.install "completions/zsh/_podman"
     fish_completion.install "completions/fish/podman.fish"
   end
 
   test do
-    assert_match "podman version #{version}", shell_output("#{bin}/podman -v")
-    assert_match(/Error: Cannot connect to the Podman socket/i, shell_output("#{bin}/podman info 2>&1", 125))
+    assert_match "podman version #{version}", shell_output("#{bin}/podman-remote -v")
```

Comment:
I guess you need `assert_match "podman-remote version #{version}"` for the test to work now.